bucket/rsync.json

```diff
@@ -0,0 +1,29 @@
+{
+    "homepage": "https://itefix.net/cwrsync",
+    "version": "6.2.3",
```

Comment:
version is usually at the top

Check https://github.com/ScoopInstaller/Main/blob/master/bucket/abc.json for the order